### PR TITLE
Adds an NNO-specific gas mask, the "nanotrasen navy officer gas mask".

### PR DESCRIPTION
**Pull request by Miraviel**
## What Does This PR Do

Adds a new NNO-specific gas mask, "nanotrasen navy officer gas mask".

The inventory icon is just a slight resprite of the warden mask.

## Why It's Good For The Game

I wanted to have a fitting gas mask for an NNO vox, but all the black masks are more grey than black - and the NNO uniforms are much darker than those. Figured all races could use this.

It is accessible via the chameleon projector.

## Images of changes

image

image

image

image

## Testing

Made the screenshots above.

## Changelog
:cl:
add: Added "nanotrasen navy officer gas mask", a gas mask specifically for NNOs.
/:cl:

### CODE REVIEW

**Comment by Miraviel** [comment]
Have to test the other races tomorrow, seeking feedback in the meantime.

**Comment by matttheficus** [comment]
If you think I'm not using this with EVERY Chameleon mask you're grossly mistake 👀

**Comment by Miraviel** [comment]
Done for all races, ready to be reviewed.

**Review by Sirryan2002 — APPROVED** [review]
Code and sprite diffs look good

File: code/modules/clothing/masks/gasmask.dm [hunk 1]
```diff
@@ -204,6 +204,13 @@
 		playsound(src.loc, 'sound/creatures/hoot.ogg', 50, 1)
 		cooldown = world.time
 
+/obj/item/clothing/mask/gas/navy_officer
+	name = "nanotrasen navy officer gas mask"
+	desc = "A durable gas mask designed for NanoTrasen Navy Officers."
+	icon_state = "navy_officer_gasmask"
+	armor = list(MELEE = 35, BULLET = 20, LASER = 20, ENERGY = 5, BOMB = 15, BIO = 0, RAD = 0, FIRE = 10, ACID = 50)
+	strip_delay = 60
```
Comment (hal9000PR) [inline code/modules/clothing/masks/gasmask.dm:212]:
```suggestion
	strip_delay = 6 SECONDS
```

**Review by hal9000PR — CHANGES_REQUESTED** [review]

**Comment by Komrad822** [comment]
> all races

Forgive me if im wrong but does Kidan need a sprite change for theirs? (would point out diona and plasmaman aswell)
Obviously this is only if there isnt a lore reason as to why they cant be NNO but idk >.>

>If you think I'm not using this with EVERY Chameleon mask you're grossly mistake
Also, same.

**Comment by Miraviel** [comment]
> > all races
> 
> Forgive me if im wrong but does Kidan need a sprite change for theirs or missing out? (would point out diona aswell) Obviously this is only if there isnt a lore reason as to why they cant be NNO but idk >.>
> 

Diona and kidan use the same mask.dmi as humans, that's why there are no racial variants for them.

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**